src/routes/SafeAppLandingPage/SafeAppLandingPage.tsx

```diff
@@ -0,0 +1,321 @@
+                        </BodyImage>
+
+                        <Button
+                          data-testid={'create-new-safe-link'}
+                          size="lg"
+                          color="primary"
+                          variant="contained"
+                          component={Link}
+                          to={OPEN_SAFE_ROUTE}
+                        >
+                          <Text size="xl" color="white">
+                            Create new Safe
+                          </Text>
+                        </Button>
+                      </>
+                    )}
+                  </>
+                ) : (
+                  <>
+                    {/* Connect your Wallet */}
+                    <StyledProvider>
+                      <Provider
+                        info={<ProviderDisconnected />}
+                        open={open}
+                        toggle={toggle}
+                        render={(providerRef) =>
+                          providerRef.current && (
+                            <Popper
+                              anchorEl={providerRef.current}
+                              style={{ zIndex: 1301 }}
+                              open={open}
+                              placement="bottom"
+                              popperOptions={{ positionFixed: true }}
+                            >
+                              <ClickAwayListener mouseEvent="onClick" onClickAway={clickAway} touchEvent={false}>
+                                <List component="div">
+                                  <ConnectDetails />
+                                </List>
+                              </ClickAwayListener>
+                            </Popper>
+                          )
+                        }
+                      />
+                    </StyledProvider>
+                  </>
+                )}
+              </UserSafeContainer>
+
+              {/* Demo Safe */}
+              <SafeDemoContainer>
+                <Title size={'xs'}>Want to try the app before using it?</Title>
+
+                <BodyImage>
+                  <Img alt="Demo" height={92} src={DemoSvg} />
+                </BodyImage>
+                {safeAppUrl && (
+                  <StyledDemoButton
+                    data-testid={'open-demo-app-link'}
+                    color="primary"
+                    component={Link}
+                    to={`${demoSafeAppsPath}?appUrl=${encodeURI(safeAppUrl)}`}
+                    size="lg"
+                    variant="outlined"
+                  >
+                    Try Demo
+                  </StyledDemoButton>
+                )}
+              </SafeDemoContainer>
+            </ActionsContainer>
+          </>
+        )}
+      </StyledCard>
+    </Container>
+  )
+}
+
+export default SafeAppLandingPage
+
+function getUserSafe() {
+  // TODO: to be implemented in https://github.com/gnosis/safe-react-apps/issues/416
+  return undefined
+}
```

Comment:
Does this need to be implemented before this is merged?

Reply:
no, this need to be implemented after this is merged and in a different PR.